tests/rules/string_mvar_julia.jl

```diff
@@ -0,0 +1,11 @@
+
```

Comment:
I don't think we need a tests/rules/ for that. a tests/patterns/julia/metavar_string.jl should be enough.